server/api/reviews.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express')
+const router = express.Router()
+const { Review, User, Order, OrderItem } = require('../db')
+
+// GET /api/reviews/:artworkId
+router.get('/:artworkId', async (req, res, next) => {
+  try {
+    const reviews = await Review.findAll({
+      where : {
+        artworkId: req.params.artworkId
+      }
+    })
+    res.send(reviews)
+  }
+  catch(err) {
+    console.log(err)
+    next(err);
+  }
+})
+
+// GET /api/reviews/user/:userId
```

Comment:
This is nitpicky, but I think the route should be /api/users/:userId/reviews to be REST-ful

Reply:
agreed! this or /api/artworks/:artworkId/reviews. either one is preferable.